schema/project-level/project-schema.json

```diff
@@ -1077,16 +1076,15 @@
         },
         "roles": {
           "title": "Party roles",
-          "description": "The party's role(s) in the project, using the open [partyRole](https://standard.open-contracting.org/profiles/ppp/latest/en/reference/codelists/#partyrole) codelist.",
+          "description": "The party's role(s) in the contracting process, using the open [partyRole](https://standard.open-contracting.org/infrastructure/{{version}}/{{lang}}/reference/codelists/#partyrole) codelist.",
           "type": [
             "array"
           ],
           "items": {
             "type": "string"
           },
           "codelist": "partyRole.csv",
-          "openCodelist": true,
-          "uniqueItems": true
```

Comment:
A couple uniqueItems are removed, I think because we ran `ocdskit schema-strict` on the schema after running borrow-schema.py. schema-strict became more strict in 0.2.8, so you might want to try with 0.2.7 if 0.2.8 adds too many extra validations.

Reply:
Fixed in https://github.com/open-contracting/infrastructure/pull/222/commits/6a656f051bee3f7839ab67c75e9174863d621413. I created an issue to remind us to run the command again before the next release https://github.com/open-contracting/infrastructure/issues/223